lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,57 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
+			which should be assumed to be already loaded (or 'provided') by the environment into which the bundle module is loaded.
+ * 	</li>
+ *	<li>
+		 <code>raw</code>: A raw section determines the set of modules that should be embedded,
+			sorts them according to their dependencies and writes them out 1:1 without any transformation or wrapping (raw). Only JavaScript sources
+				can be embedded in a raw section.
+	</li>
+ *	<li>
+		 <code>preload</code>: A preload section packages resources that should be stored in the preload cache in the client.
+			They can embed any textual resource type (JavaScript, CSS and Theme-Parameter files) that the bundling supports.
+			UI5 modules are wrapped into a 'sap.ui.predefine' call. Other JavaScript modules will be embedded into a 'jQuery.sap.registerPreload' call, unless the
+			asynchronous ui5loader is used. With the ui5loader 'sap.ui.require.preload' is used for other modules.
+	</li>
+ *	<li>
+		 <code>require</code>: A require section is transformed into a sequence of jQuery.sap.require calls. It only has text content, no nested options.
+			The text content must contain a comma or whitespace separated list of module names or patterns. The list will be resolved like an include pattern list
+				in any of the other sections and for each of the resolved modules, a jQuery.sap.require will be created.
```

Comment:
This also depends on whether the ui5loader is available. In that case we use `sap.ui.requireSync`.

Reply:
Done